content/influxdb/v2.2/users/recover-credentials.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Recover user credentials
+seotitle: Recover InfluxDB user credentials
+description: Recover InfluxDB user credentials using the influx CLI.
+menu:
+  influxdb_2_2:
+    name: Recover credentials
+    parent: Manage users
+weight: 106
+products: [oss]
+---
+
+Use the `influx` command line interface (CLI) to recover user credentials, view all users in the system, or update a password.
+
+### Recover all users
+
+To recover all users in the system, run the following:
```

Comment:
should we clarify all users on disk or other distinction to be made here?

Reply:
No distinction I think. I would say "to list existing users in the system" rather than recover